### PR TITLE
Update information about AWS setting values

### PR DESCRIPTION
**Pull request by Tequilac**

### CODE REVIEW

**Comment by mkrawczyk-avs** [comment]
thanks for the update!

File: docs/AWS_Integration_Guide/Configuring_AWS_integration.md [hunk 1]
```diff
@@ -51,35 +51,16 @@ Then follow the steps below:
...
-              ![Copy data plane endpoint address](images/dataplane.png "Copy data plane endpoint address")
-            - In Coiote DM, go to the **Profiles** tab of your integration group and paste the result as the value for `AWSdataPlaneEndpointAddress`.
-            - Append `:8443` port to the pasted value.
-            - Click **Save**.
+- Go to **Device groups**, select your custom integration group (or the **AWSiotCoreCertAuth** group, depending on the previous step) and go to **Profiles**. Complete the **AWS** setting value:
+  - Open your command line and run the following command:
+           ```
+           aws iot describe-endpoint --endpoint-type iot:Data-ATS
+           ```
+  - Copy the returned result.
```
Comment (mikegpl) [inline docs/AWS_Integration_Guide/Configuring_AWS_integration.md:59]: Is the returned result just an URL, or rather sth else like JSON?

File: docs/AWS_Integration_Guide/Configuring_AWS_integration.md [hunk 2]
```diff
@@ -51,35 +51,16 @@ Then follow the steps below:
...
+  ![Copy data plane endpoint address](images/dataplane.png "Copy data plane endpoint address")
+  - In Coiote DM, go to the **Profiles** tab of your integration group and paste the result as the value for `AWSdataPlaneEndpointAddress`.
+  - Append `:8443` port to the pasted value.
```
Comment (mikegpl) [inline docs/AWS_Integration_Guide/Configuring_AWS_integration.md:62]: Can't we do this automatically in the tasks?

File: docs/AWS_Integration_Guide/Configuring_AWS_integration.md [hunk 3]
```diff
@@ -102,6 +83,8 @@ To add the resources needed for the integration to your AWS services:
     ```
     python3 -m pip install -r lwm2mOperation/requirements.txt --target lwm2mOperation/
     python3 -m pip install -r certificates/requirements.txt --target certificates/
+    python3 -m pip install -r createThingType/requirements.txt --target createThingType/
```
Comment (mikegpl) [inline docs/AWS_Integration_Guide/Configuring_AWS_integration.md:86]: I see the requirements were added but I don't see the two new lambdas, do I understand correctly that it should be included in (some) revision?

File: docs/AWS_Integration_Guide/Configuring_AWS_integration.md (hunk 1 above)
Comment (Tequilac) [reply docs/AWS_Integration_Guide/Configuring_AWS_integration.md:59]: JSON with a single value

File: docs/AWS_Integration_Guide/Configuring_AWS_integration.md (hunk 2 above)
Comment (Tequilac) [reply docs/AWS_Integration_Guide/Configuring_AWS_integration.md:62]: we could, but we've decided not to write the same port in every task

File: docs/AWS_Integration_Guide/Configuring_AWS_integration.md (hunk 3 above)
Comment (Tequilac) [reply docs/AWS_Integration_Guide/Configuring_AWS_integration.md:86]: look inside your heart. maybe the true lambdas were the friends you made along the way?

**Comment by mikegpl** [comment]
LGTM

**Pull request merged**